README.md

```diff
@@ -88,28 +90,28 @@ Now, all that's left to do is `cargo run`.
 
 Congrats! You've executed your first `JavaScript` using `Boa`!
 
-## Documentation
+## 🔎 Documentation
 
 For more information on `Boa`'s API. Feel free to check out our documentation.
 
 [**API Documentation**](https://docs.rs/boa_engine/latest/boa_engine/)
 
-## Conformance
+## 🏅 Conformance
 
 To know how much of the _ECMAScript_ specification does Boa cover, you can check out results
 running the _ECMASCript Test262_ test suite [here](https://boajs.dev/conformance).
 
-## Contributing
+## 🪚 Contributing
 
 Please, check the [CONTRIBUTING.md](CONTRIBUTING.md) file to know how to
 contribute in the project. You will need Rust installed and an editor. We have
 some configurations ready for VSCode.
 
-### Debugging
+### 🐛 Debugging
 
 Check [debugging.md](./docs/debugging.md) for more info on debugging.
 
-### Web Assembly
+### 🕸 Web Assembly
```

Comment:
question: Why a spiderweb? Is that used elsewhere for web assembly?

Reply:
> question: Why a spiderweb? Is that used elsewhere for web assembly?

I couldn’t come up with a better option... It kind of fits since it’s related to WebAssembly. :sweat_smile:
I haven't seen it anywhere, most repos that I've seen that use emojis for sections are a lot of NeoVim packages.